config/server-config.yaml.example

```diff
@@ -89,3 +89,13 @@ authz:
   auth:
     # Set to token for production
     method: none
+
+# Configuration for the healthcheck functionality
+# Defaults to disable if not defined
+#marketplace:
+#  enabled: true
+#  profile_name: stacklok-health-check
+#  bundle:
+#    filepath: ./bundle/healthcheck.tar.gz
```

Comment:
to indicate we'll eventually support other means of installation, should we do a driver-like approach?

```yaml
bundle:
  type: tar
  tar:
    filepath: ./bundle/healthcheck.tar.gz
  namespace: ...
```